ext/web/02_event.js

```diff
@@ -113,7 +112,7 @@ function setStopImmediatePropagation(
 function hasRelatedTarget(
   event,
 ) {
-  return ReflectHas(event, "relatedTarget");
+  return !!event.relatedTarget;
```

Comment:
This is not the same thing. If `relatedTarget` coerces to `null`, this falsely returns false. Use `"relatedTarget" in event`

Reply:
I used `in` initially but there`s a lint rule against `in` usage. So went with object lookup because in these changes there wasn't any difference given the possible values and usage of `hasRelatedTarget`.

Being honest, couldn't find any reasoning as to why `Reflect.has` was being used. Looked in Node.js implementation and the spec. `relatedTarget` as far as I know should be either `null` or a target. So the current implementation with `Reflect.has` was returning `true` for `relatedTarget === null` which IMO is not correct.

Reply:
> Can we just return false here? We don't have any DOM events.

I was actually going to change this later with 

```
const eventRelatedTarget = eventImpl.relatedTarget ?? null;
```
And most likely `?? null` is not even needed.

There are a lot of properties and paths that are most likely never used currently, but can't find examples of code that is supposed to used those branches so it is hard to know whether it'll break things or not, which is why I'm only making small changes for now.

Reply:
If no WPTs break, it is safe to change for sure. If they do, it _may_ be safe.

Reply:
Also: relatedTarget should always be the EventTarget that the event was dispatched on, I think.

This check can be `!== null` I think

Reply:
Check is actually not needed given possible `relatedTarget` values.  Removed `hasRelatedTarget`,